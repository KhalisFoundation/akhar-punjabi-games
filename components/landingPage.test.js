import React from 'react';
import renderer from 'react-test-renderer';
import HomeScreen from './homeScreen/landingPage';

describe('<HomeScreen />', () => {
  it('has 1 child', () => {
    const tree = renderer.create(<HomeScreen />).toJSON();
    expect(tree.children.length).toBe(1);
  });
  it('renders correctly', () => {
    const tree = renderer.create(<HomeScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});