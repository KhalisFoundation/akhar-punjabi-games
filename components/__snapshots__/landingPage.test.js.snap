// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeScreen renders without crashing - with user 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;

exports[`HomeScreen renders without crashing 1`] = `null`;
